extensions/jms/src/main/java/org/mule/extensions/jms/internal/message/JmsMessageUtils.java

```diff
@@ -70,11 +70,6 @@ public static Message toMessage(Object object, Session session) throws JMSExcept
     } else if (object instanceof InputStream) {
       return inputStreamToMessage((InputStream) object, session);
 
-      // TODO MULE-12270 : SDK Should provide a generic way to handle CursorStreamProviders
```

Comment:
you removed this but didn't add the necessary code in the runtime to solve this problem